### DISCUSSION TITLE
请问如何导出fastGPT的对话记录呢

### DISCUSSION THREAD

**Discussion by Terminaldienst** [labring/FastGPT#912] [Q&A]
需要完整包含提示词知识库上下文用户提问的input，以及对应的output。
如果不能直接导出，请问这一块是在数据库中如何存储的呢，希望开发者大大不吝赐教。

**Comment by c121914yu** [comment]
> 需要完整包含提示词知识库上下文用户提问的input，以及对应的output。
> 如果不能直接导出，请问这一块是在数据库中如何存储的呢，希望开发者大大不吝赐教。

chats 和 chatitems 表中

**Reply by Terminaldienst** [reply]
请问通过API调用的完整的（包含提示词和知识）对话也可以查到吗
